src/editor/editor.js

```diff
@@ -104,6 +104,19 @@ export default class Editor {
 		 */
 		this.t = this.locale.t;
 
+		/**
+		 * Indicates editor initialization status.
+		 *
+		 * The following statuses are available:
+		 * * initializing -  during initialization chain
```

Comment:
A person reading this documentation will not know that `create()` has a chain of promises inside it. Use the terms which the audience know.